CSharp/Blurhash.Core/MathUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Blurhash.Core
+{
+    /// <summary>
+    /// Utility methods for mathematical calculations
+    /// </summary>
+    public static class MathUtils
+    {
+        /// <summary>
+        /// Calculates <c>Math.Pow(base, exponent)</c> but retains the sign of <c>base</c> in the result.
+        /// </summary>
+        /// <param name="base">The base of the power. The sign of this value will be the sign of the result</param>
+        /// <param name="exponent">The exponent of the power</param>
+        public static double SignPow(double @base, double exponent)
+        {
+            return Math.Sign(@base) * Math.Pow(Math.Abs(@base), exponent);
+        }
+
+        /// <summary>
+        /// Converts an sRGB input value (0 to 255) into a linear double value
+        /// </summary>
+        public static double SRgbToLinear(int value) {
```

Comment:
I admit, I took this from another port of the algorithm, but I don't understand why it is neccessary.
Why can't I just divide the value by 255.0?

Reply:
Because most RGB colours you encounter, such as in a bitmap, will be in sRGB colour space, which is not linear. However, the algorithm works with linear values so they have to be converted back and forth. Check out https://en.wikipedia.org/wiki/SRGB for a lot more detail.

Reply:
Thanks, I read up on that :)